tensilelite/Tensile/TensileInstructions/Math.py

```diff
@@ -72,8 +72,13 @@ def vectorStaticDivideAndRemainder(qReg, rReg, dReg, divisor, tmpVgprRes: Option
         module.add(VLShiftRightB64(dst=vgpr(tmpVgpr,2), shiftHex=hex(shift), src=vgpr(tmpVgpr,2), comment=dComment))
         module.add(VMovB32(dst=vgpr(qReg), src=vgpr(tmpVgpr), comment=dComment))
         if doRemainder:
-            module.add(VMulLOU32(dst=vgpr(tmpVgpr), src0=vgpr(qReg), src1=hex(divisor), comment=rComment))
+            if divisor <= 64 and divisor >= -16:
```

Comment:
what does the negative mean?

Reply:
[-16, 64] is the supported literal range. This part is written with reference to method **vectorStaticRemainder**.